Cargo.toml

```
@@ -22,7 +22,7 @@ members = [
 resolver = "2"
 
 [patch.crates-io]
-xtra = { git = "https://github.com/Restioson/xtra", rev = "7e9fccc1e02dae3df4ba99126e6f1a0059673d7e" } # Unreleased
+xtra = { git = "https://github.com/Restioson/xtra", branch = "span-late-create" } # TODO Unreleased
```

I have created a stable reference until xtra#120 gets in (and updated the comment - TODO comments that ain't actionable are super easy too address later)

```suggestion
xtra = { git = "https://github.com/comit-network/xtra", branch = "span-late-create" } # TODO Switch back to xtra/master after https://github.com/Restioson/xtra/pull/120 gets merged
```